### ISSUE TITLE
USA - PP, Stability and WS are not increase after USC beats the African rebel states

### ISSUE THREAD

**Issue by Kyklosian** [TNODevs/TNO-Bug-Tracker#1044]
### Operating System

Windows 10 Pro

### HOI4 version

Windows 10 Pro

### TNO version

1.2.1c

### Do you have other mods in use?

None

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

Save game: [USA_USC_wins_the_war.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/9626279/USA_USC_wins_the_war.zip)

### Describe the bug

Despite USC managed to win against the African rebels, the political power, stability, and war support do not increase or does not receive the event about handling the CAR situation because these were been greatly debuffed when selecting the 'Chaos in the CAR' world news event. 

### Steps to reproduce / Cause of the issue

1. The Central African Republic is formed.
2. Conference fails (always fails, unfortunately despite the administrations and military stations are high status).
3. USC chooses to warpath focuses.
4. Chaos in CAR.
5. USC managed to defeat the rebels.

### Expected behaviour

I don't know what causes this.

### Screenshots

![Untitled_1](https://user-images.githubusercontent.com/71108541/191762504-c2712eb1-9929-4302-ba7d-7bb3f464a3d2.png)

![Untitled_2](https://user-images.githubusercontent.com/71108541/191762512-34ad7642-b42d-4b0f-a508-d6ab0561e3d8.png)

### Additional information

I don't have any additional information.

**Comment by Jack121Q** [comment]
I don't think that could be called a win anyhow. I don't think it warrants adding a buff.
